src/create-glamorous.js

```diff
@@ -112,43 +75,18 @@ function createGlamorous(splitProps) {
             ...toForward,
             className,
           })
-        }
-      }
+        },
+        {noWarn: true, createElement: false},
```

Comment:
Brief explanation:
- `noWarn`: If there's no theme context, don't log to the console. Logging was the default behavior (because why would you use `withTheme` if there was no theme in the context?) However in our case you can totally use glamorous components without the theme context so we don't want to warn about that. So we opt-out. Default behavior remains the same (no breaking change).
- `createElement`: This is where things get a little tricky. In our case, `GlamorousComponent` is now actually just a regular function that's called by the `ThemedComponent`. This way we avoid adding another layer in the React Tree and we don't actually need lifecycle hooks for this component anyway so we're covered. Default case is `true` so that is unchanged (no breaking change).